src/BinaryParsers/ElfBinary/Dwarf/DwarfAttributeValue.cs

```diff
@@ -77,6 +77,10 @@ public class DwarfAttributeValue
         /// </summary>
         public object Value { get; set; }
 
+        // Gets or sets a property that temporarily stores an offset that 
+        // will beused to retrieve the actual attribute value.
+        public ulong? Offset { get; set; }
```

Comment:
>Offset [](http://example.com/codeflow?start=22&length=6)

So there's a core design issue, that when parsing certain attribute specifications, there may be information elsewhere in the current set of specifications that are required to resolve an actual attribute value.

For example, a string attribute may include an offset in the literal attribute data stream. This offset can't be interpreted unless/until a different attribute spec provides a base location for that offset.

With the current code, what we do is to populate this 'Offset' property with the interim attribute spec data. We need to figure out later how best to handle post-processing to resolve all these offsets into actual values.

It's not clear that resolving all these is strictly required for our binary analysis. It would be for any comprehensive DWARD5 parsing component, of course.